sree.cfg.sample

```diff
@@ -1,3 +1,4 @@
 RGW_CIVETWEB_PORT="RGW_CIVETWEB_PORT_VALUE"
+RGW_ADDRESS="RGW_ADDRESS_VALUE"
```

Comment:
What's the default value?

Reply:
@leseb Should I specify it? If yes how?

Reply:
Well, I'd like to keep the existing behavior of `127.0.0.1`, so somehow this needs to be present.

Reply:
@leseb In ceph-container I set it to `127.0.0.1` as default. https://github.com/ceph/ceph-container/pull/1797

Reply:
Because of the `sed` that is using in ceph-container I can't set `127.0.0.1` as a value here. Maybe it's better to change the config generator there so we can set the default values here!